package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frontend-component-footer-mogc",
-  "version": "14.2.1",
+  "version": "14.3.0",
```

Comment:
I didn't really question this last time we updated, but how did this get to 14?

Reply:
I was probably intending to match the Tutor release (which matched the Open edX release). So 14 would have been for the 14th letter of the alphabet, N(utmeg).